sharding-jdbc/sharding-jdbc-core/src/test/java/io/shardingsphere/dbtest/fixture/ConstantKeyGenerator.java

```diff
@@ -18,9 +18,17 @@
 package io.shardingsphere.dbtest.fixture;
 
 import io.shardingsphere.core.keygen.KeyGenerator;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.Properties;
 
 public final class ConstantKeyGenerator implements KeyGenerator {
     
+    @Getter
```

Comment:
Move @Getter and @Setter to class level if all fields need getter and setter, even there is only one field.